recipes-extended/jailhouse/jailhouse/0001-scripts-tools-switch-to-python3.patch

```diff
@@ -0,0 +1,81 @@
+From c9fc6bbd535d0d92578cf3258c6db052ec6094dd Mon Sep 17 00:00:00 2001
+From: Cristinel Panfir <cristinel.panfir@nxp.com>
+Date: Tue, 28 Apr 2020 13:59:21 +0300
+Subject: [PATCH] scripts/tools: switch to python3
+
+Signed-off-by: Cristinel Panfir <cristinel.panfir@nxp.com>
+---
```

Comment:
Upstream-Status is missing in the patch file. Is it a backport or this patch committed upstream>

Reply:
It's not a backport or upstream submitted, the origin has not been updated to support newer user space apps, which this patch is referring to.
OK, it can be set to Pending.